apps/tx-builder/src/lib/getAbi.ts

```diff
@@ -166,6 +169,19 @@ const getGatewayBaseUrl = (chain: string) => {
   }
 }
 
+const getScanAPIBaseURL = (chain: string) => {
+  switch (chain) {
+    case SUPPORTED_CHAINS.LINEA:
+      return 'https://api.lineascan.build'
+    case SUPPORTED_CHAINS.LINEA_TESTNET:
+      return 'https://api-testnet.lineascan.build'
+    default:
+      throw new Error(
+        `[getScanAPIBaseURL]: There is no scan API for ${chain}, therefore we cannot get the contract abi from it.`,
+      )
```

Comment:
@msvstj I understand, that you were trying to keep the same structure as in the existing functions, but that's not the point here. This error will be visible on other networks except these 2, but there are more than 20 of them

Reply:
Alright, makes sense not to include it as it may be required to a small amount of networks in the future. Will remove !